rts/lra/hotel-service/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fabric8/java-jboss-openjdk8-jdk:1.2.3
+FROM registry.access.redhat.com/ubi8/openjdk-21:1.18
```

Comment:
Same image of lra coordinator quarkus : https://github.com/jbosstm/lra-coordinator-quarkus/blob/main/src/main/docker/Dockerfile.jvm#L80